skia-safe/src/effects/merge_image_filter.rs

```diff
@@ -0,0 +1,35 @@
+use crate::prelude::*;
+use crate::{ImageFilter, ImageFilterCropRect};
+use skia_bindings::{C_SkMergeImageFilter_Make, SkImageFilter};
+use std::convert::TryInto;
+
+pub enum MergeImageFilter {}
+
+impl MergeImageFilter {
+    #[allow(clippy::new_ret_no_self)]
+    pub fn new<'a, CR: Into<Option<&'a ImageFilterCropRect>>>(
+        filters: &[&ImageFilter],
```

Comment:
Can we somehow accept more types here, like a `Vec<&ImageFiler>`.

Reply:
an `IntoIterator` trait would be too much, and I can't think of anything in between.